WTWSMS/decisions/realm_decisions.txt

```diff
@@ -2028,26 +2061,6 @@ decisions = {
 					value = off
 				}
 			}
-			
-			OR = {
-				ai = no
-				AND = {
-					trait = proud
-					prestige = 5000
-				}
-				AND = {
-					trait = ambitious
-					prestige = 5000
-				}
-				AND = {
-					realm_size = 70
-					any_demesne_title = {
-						count = 4
-						tier = DUKE
-					}
-				}
-				prestige = 15000
-			}
```

Comment:
The code is from vanilla. Historically custom realms didn't form constantly. That is why there is a need for some restrictions. The AI isn't meant to seek to always blobb: many borders remained static.